docs/requirements.txt

```diff
@@ -12,3 +12,5 @@ jax
 dm-haiku
 optax
 rdkit-pypi
```

Comment:
Also, do you need torch sparse in docs? It seems only torch geometric is used in docstrings.

Reply:
> Also, do you need torch sparse in docs? It seems only torch geometric is used in docstrings.

torch geometric requires sparse and scatter

Reply:
> Try adding `--extra-index-url https://data.pyg.org/whl/torch-1.12.0+cpu.html` for installing torch-geometric because the docs [here](https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html) for `pip` adds the extra index url.

Thanks a lot. I have made the commit. Lets see.

Reply:
For torch sparse, inplace of the `torch-sparse>=0.6.14`, replace it directly with the link to the following whl file: https://data.pyg.org/whl/torch-1.12.0%2Bcpu/torch_sparse-0.6.14-cp38-cp38-linux_x86_64.whl . I think this will fix it.

Reply:
This worked! Thanks a lot for the help.